src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let number = 0;
+
+  if (days < 3) {
+    number = days * 40;
+  } else if (days >= 3 && days < 7) {
```

Comment:
Use `if` with `return` to simplify the logic and avoid unnecessary `else if` blocks. For example, you can return the calculation directly in the first `if` block.